flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/rest/util/SqlGatewayRestEndpointConfigOptions.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.gateway.rest.util;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.configuration.ConfigOption;
+
+import static org.apache.flink.configuration.ConfigOptions.key;
+
+/** Options to configure {@link org.apache.flink.table.gateway.rest.SqlGatewayRestEndpoint}. */
+@PublicEvolving
+public class SqlGatewayRestEndpointConfigOptions {
+
+    public static final String GATEWAY_ENDPOINT_PREFIX = "sql-gateway.endpoint.";
+
+    public static final ConfigOption<String> BIND_ADDRESS =
+            key("rest.bind-address")
+                    .stringType()
+                    .defaultValue("0.0.0.0")
+                    .withDescription("The address that the rest endpoint binds itself.");
+
+    public static final ConfigOption<String> BIND_PORT =
+            key("rest.bind-port")
+                    .stringType()
+                    .defaultValue("8083")
```

Comment:
Sql gateway will support this feature. This is because `SqlGatewayRestEndpoint` directly inherits `RestServerEndpoint`, which is responsible for the analysis of related configurations. The process of parsing the configuration is the same as `RestServerEndpoint` in runtime module.